Library/Formula/fleetctl.rb

```diff
@@ -15,6 +15,16 @@ class Fleetctl < Formula
 
   depends_on "go" => :build
 
+  def caveats; <<-EOS.undent
+    fleetctl versions >= 0.10.0 use the fleet API instead of etcd by default.
+
+    If you want to revert to previous behavior, you can use the --driver flag:
+      fleetctl --driver=etcd <cmd>
+    Or you can set it with FLEETCTL_DRIVER:
+      export FLEETCTL_DRIVER="etcd"
+    EOS
+  end
```

Comment:
Can we try and slim down the number of lines here? I also wonder if we even need these given they aren't Homebrew-specific.